en_ewt-ud-test.conllu

```diff
@@ -11256,27 +11256,27 @@
 # text = - PPA Guaranty.doc
 1	-	-	PUNCT	NFP	_	2	punct	2:punct	_
 2	PPA	ppa	NOUN	GW	_	0	root	0:root	_
-3	Guaranty.doc	guaranty.doc	X	NN	Number=Sing	2	goeswith	2:goeswith	_
+3	Guaranty.doc	guaranty.doc	X	NN	Number=Sing	2	flat	2:flat	_
```

Comment:
Not sure what is best for the UPOS and features in these filenames. If `Number` is specified does that imply `NOUN` rather than `X`?